### ISSUE TITLE
[Console] Integrations custom JSON input box keeps messing up input

### ISSUE THREAD

**Issue by ScruffR** [spark/cloud#49]
With some indentation and "special" characters, the carret location and the actual input get gradually misaligned.
Tested in most recent Chrome browser on Win10 Pro 64bit
When I place the carret directly after closing double quote of `"{{capacity}}"`
![image](https://user-images.githubusercontent.com/6604878/43124906-c850d92a-8f28-11e8-8cfa-77a2c63a1aa6.png)
But when I type a letter there it appears somewhere else (find the `|` for illustration)
![image](https://user-images.githubusercontent.com/6604878/43124888-bb385650-8f28-11e8-8764-92874c98103a.png)

That's particularly annoying, when - for example - you type an opening double quote and the auto-complete adds the closing one, because then you see the carret jump two positions, but when you hit backspace both of them disappear again, since the actual input positiono has only jumped one, and you are actually deleting the opening double quote.

**Comment by monkbroc** [comment]
That sounds annoying. I'm not sure how to reproduce this. It's not something I've seen before.

Does this continue after refreshing the page? Can you make this happen by create a new webhook and typing specific characters in the field?

**Comment by ScruffR** [comment]
Yup, that's a persistent thing for me, even after rebooting the machine, chaning font size or zoom rate in the browser.

It seems to be in connection with auto-indent and auto-completion. But I can do some more tests.

**Comment by monkbroc** [comment]
I wonder if it has to do with the language of the OS. Is your Windows in German?

**Comment by ScruffR** [comment]
In deed it is, but I can check what happens if I change the default language.
I can also try another machine.

Update: I've tried on another machine, and there the issue is not reproducable.
But on my MS Surface Pro (2017) the issue persists regardless of language and it's also not confined to Chrome but also happens with Firefox.

No idea what's wrong, but since this seems a rather isolated issue with my tablet, I guess there isn't an easy cure.
